examples/js/loaders/GLTFLoader.js

```diff
@@ -25,10 +25,13 @@ THREE.GLTFLoader = ( function () {
 
```

Comment:
Could you define `this.path` and `this.resourcePath`, as null or undefined or `''`, in the GLTFLoader constructor? I find it helpful to have all instance properties listed in one place.

Reply:
I think `.path` (and `.resourcePath`) in loader is `undefined` or strings, and the default is `undefined`. As far as I know most of loaders don't explicitly define the default in the constructor (because no define is `undefined`). I wanted to follow this manner for the consistency. I think if you'd like to explicitly define in the constructor, maybe you'd better to first open an issue and discuss as we did for `setPath()`?

Reply:
Even if they remain undefined that's OK, I'd just like for the list of properties to appear in the constructor. For example:

```js
this.path;
this.resourcePath;
```

Reply:
I meant, most of loaders don't do that. I prefer consistency with other loaders. I wanna add those lines not only to glTF loaders but also other loaders, including file loader, if we do. So I want to first open an issue, or we could use #12942, to discuss.

Reply:
I think that's a "code style" question more than a behavior question, and I don't think it's realistic to keep the same code style across all loaders without investing a lot of unnecessary time. I'd prefer to be able to keep a clean code style in GLTFLoader without the burden of cleaning up every other loader, and declaring instance-level properties in one place is something I consider necessary for clean code.

Reply:
I still prefer the consistent code style in this case. If you want to do that, would you please open another PR?

Reply:
Ok, will do. 👍

Reply:
Sure, that's fine yes. :)